Mathlib/Data/Complex/Basic.lean

```diff
@@ -265,6 +265,18 @@ theorem I_mul_re (z : ℂ) : (I * z).re = -z.im := by simp
 
 theorem I_mul_im (z : ℂ) : (I * z).im = z.re := by simp
 
+theorem mul_I_re_ofReal (x : ℝ) : (x * I).re = 0 := by
+  rw [mul_I_re, ofReal_im, neg_zero]
+
```

Comment:
```suggestion
@[simp]
theorem mul_I_re_ofReal (x : ℝ) : (x * I).re = 0 := by
  rw [mul_I_re, ofReal_im, neg_zero]

```
and the same for the other three.

Reply:
@dupuisf  The linter complains:
```
/- The `simpNF` linter reports:
SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%20form -/
-- Mathlib.Data.Complex.Basic
Error: ././././Mathlib/Data/Complex/Basic.lean:268:1: error: Complex.mul_I_re_ofReal simp can prove this:
  by simp only [Complex.mul_re,
 Complex.ofReal_re,
 Complex.I_re,
 @mul_zero,
 Complex.ofReal_im,
 Complex.I_im,
 @mul_one,
 @sub_self]
One of the lemmas above could be a duplicate.
If that's not the case try reordering lemmas or adding @[priority].
```
Do you think these lemmas are worth having without `@[simp]` or should I remove them altogether?

Reply:
Ah, that means that simp can already prove these without adding them to the simp set. So you can just remove the attributes. (Or remove the lemmas altogether -- up to you I'd say).